### PR TITLE
Update mtd.md to include the fast track benefits

### PR DESCRIPTION
**Pull request by itsmukta**

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/mtd.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ This helps reduce potential issues that could arise while rolling out the servic
 
 - Alerts appear in dashboard: Verify that alerts for above detection tests appear on the [security console](https://security.microsoft.com).
 
+Need help in deploying or configuring Defender for Endpoint on Android & ioS ? If you have at least 150 licenses for  the product, use your [FastTrack benefits](https://learn.microsoft.co/microsoft-365/fasttrack/introduction). Refer [here] (https://fasttrack.microsoft.com/)for more details about FastTrack.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mtd.md]:
```suggestion
Need help in deploying or configuring Defender for Endpoint on Android & iOS? If you have at least 150 licenses for the product, use your [FastTrack benefits](/microsoft-365/fasttrack/introduction). Learn more about FastTrack [here](https://fasttrack.microsoft.com/).
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @itsmukta, kindly commit the suggested changes. Many thanks

**Review by itsmukta — COMMENTED** [review]
updated with the suggestions

**Comment by itsmukta** [comment]
@JohanFreelancer9 : made the changes. pls review.

File: microsoft-365/security/defender-endpoint/mtd.md [hunk 2]
```diff
@@ -112,6 +112,8 @@ This helps reduce potential issues that could arise while rolling out the servic
 
 - Alerts appear in dashboard: Verify that alerts for above detection tests appear on the [security console](https://security.microsoft.com).
 
+Need help in deploying or configuring Defender for Endpoint on Android & ioS ? If you have at least 150 licenses for  the product, use your [FastTrack benefits](/microsoft-365/fasttrack/introduction). Learn more about FastTrack [here] (https://fasttrack.microsoft.com/).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mtd.md]:
```suggestion
Need help in deploying or configuring Defender for Endpoint on Android & iOS? If you have at least 150 licenses for the product, use your [FastTrack benefits](/microsoft-365/fasttrack/introduction). Learn more about FastTrack [here] (https://fasttrack.microsoft.com/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@itsmukta, three more, please.

**Pull request merged**